.gitignore

```diff
@@ -1,2 +1,4 @@
 src.zip
 temp
+.idea
```

Comment:
Surely this should be ignored by your `.gitignore_global` file

Reply:
You're absolutely right. I'll remove this commit from the PR.